configure

```diff
@@ -341,6 +349,23 @@ CXXCMD=`./scripts/find_compiler.sh cxx --ccache=$with_ccache --script-dir=${PWD}
 echo "Setting C compiler to $CCCMD."
 echo "Setting C++ compiler to $CXXCMD."
 
+if [[ $with_clangtidy == 1 ]]; then
+  clangtidy_path=`which clang-tidy || echo ''`
+  
+  if ! [ -f $clangtidy_path ]; then
```

Comment:
bash style should be `[[ ! ]]`

Reply:
better to wrap `$clangtidy_path` with quotes. Otherwise, the shell is free to expand what's referenced.